docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -20,6 +20,25 @@ This is a list of things to check if you want to update from a previous version
 - Optionally check also accessory files like `.eslinrc`, if you want to keep aligned with lint standards.
 - Follow the instructions below, in order, from your version to the one you want to update to.
 
+## Migration from 2024.02.00 to 2025.01.00
+
+### Add Favorite plugin to localConfig.json
+
+The new Favorite plugin should be added inside the plugins `maps` section of the localConfig.json to visualize the button on the resource cards
+
+```diff
+{
+    "plugins": {
+        ...,
+        "maps": [
+            ...,
++           { "name": "Favorites" }
+        ],
+        ...
+    }
+}
+```
+
```

Comment:
I think we missed some migrations here:
- ResourcesFiltersForm
- Other changes on maps and other pages.

Should we put them in another PR?

```suggestion
The new Favorite plugin should be added inside the plugins `maps` section of the `localConfig.json` to visualize the button on the resource cards

```diff
{
    "plugins": {
        ...,
        "maps": [
            ...,
+           { "name": "Favorites" }
        ],
        ...
    }
}
```

```